modules/ROOT/pages/_partials/afm-concepts.adoc

```diff
@@ -0,0 +1,73 @@
+// Reused in index.adoc and specific feature topics
+
+// tag::afm-monitors[]
+
+A monitor is a repeating schedule of one or more tests. Monitors might also be referred to as test suites. 
+
+// end::afm-monitors[]
+
+// tag::afm-tests[]
+
+Tests are run in monitors on a defined schedule. Test cases are defined using Behavior Driven Development (BDD) language, a common, declarative, and easy-to-use behavior-description language. 
+
+// end::afm-tests[]
+
+// tag::afm-bdd[]
+
+The BDD language is used to write tests. It is based on DataWeave and follows the “given-when-then” approach to describe behavior in terms of conditions and expected outcomes.
```

Comment:
If you remove the "The BDD language is," you can say "A DataWeave-based language that follows...

Reply:
Same as mentioned previously. Sometimes the BDD is generated.